block/block.py

```diff
@@ -49,9 +50,13 @@ def block_timestamp(self):
         # gets the time and sets timestamp
         return str(date.datetime.now()).encode('utf8')
 
+    def genesis_timestamp(self):
+        # gets the time and sets timestamp
+        self.timestamp = str("2017-10-08 13:18:06.810644").encode('utf8')
+
     def merkle_root(self):
         # calculates the merkle root and sets it as the blocks merkle_root
-        return 0000000000000000000000000000000000000000000000000000000000000000
+        return 0

```

Comment:
did you merge master into your branch? This should have called the merkle helper function